website/src/pages/cwkw/cwkw-layout.css.ts

```diff
@@ -39,11 +39,31 @@ export const subHeader = style([baseLayout.subHeader, { fontSize: "larger" }])
 
 export const siteTitle = style([baseLayout.siteTitle, { flex: 1 }])
 
-export const siteLink = style([baseLayout.siteLink, { display: "flex" }])
+export const siteSubtitle = style([
+  media(mediaQueries.medium, marginY(vspace.half)),
+  marginY(vspace.eighth),
+  {
+    flex: 1,
+    color: colors.primaryText,
+    vars: {
+      [linkColor]: "white",
+      [colors.focus]: "white",
+    },
+  },
+])
+
+export const siteLink = style([
+  baseLayout.siteLink,
+  {
+    display: "flex",
+  },
+])
 
 export const contentContainer = style([
   baseLayout.contentContainer,
-  { flex: 1 },
+  {
+    display: "block",
+  },
 ])
```

Comment:
This doesn't seem right to me. I think we wanted to keep the flex property and get rid of display block, since this is applied to a block level element?